### ISSUE TITLE
Running on device fails. Tried manual linking and cocoapods

### ISSUE THREAD

**Issue by davidfloegel** [ivpusic/react-native-image-crop-picker#218]
I know this has been asked before but I still can't get it to run so I'm posting again. sorry!

### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.11.12
- react-native v0.39.2

### Platform
Tell us to which platform this issue is related
- iOS

### Expected behaviour

It should build

### Actual behaviour

It used to run fine on the simulator but as I wanted to run the app on my phone I tried linking the frameworks via cocoapods but that didn't work. Manual linking did not work either.

### Attachments

```
ld: warning: directory not found for option '-L/Users/david/Library/Developer/Xcode/DerivedData/app-cqukxjcsopckzmchwnyeizuzoysy/Build/Products/Debug-iphonesimulator/QBImagePickerController'
ld: warning: directory not found for option '-L/Users/david/Library/Developer/Xcode/DerivedData/app-cqukxjcsopckzmchwnyeizuzoysy/Build/Products/Debug-iphonesimulator/RSKImageCropper'
ld: library not found for -lQBImagePickerController
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by bluedaniel** [comment]
I've got the same issue and can't build this for release either.

**Comment by theseansy** [comment]
Not sure if @bluedaniel had the same issue, but I can run debug builds on device, but making release builds fail most of the time with a `RSKImageCropper/RSKImageCropper.h not found` error from `imageCropPicker.xcodeproj/ImageCropPicker.h`.

To fix this, I clean derived data, re-install pods/npm modules, and then it still has the same error about half of the time.

**Comment by tobycox** [comment]
@theseansy Yeah, I get the same issue. Do an Archive release one day, then a few days later try again and it's broken. The classic combo of:
1. `pod install`
2. `react-native link`
3. Xcode clean and restart

usually fixes it, but it's super frustrating.

**Comment by theseansy** [comment]
@tobycox - I ended up going with just using the [frameworks and manually linking](https://github.com/ivpusic/react-native-image-crop-picker#non-cocoapods-users) as an embedded binary vs. using cocoapods to install RSKImageCropper and QBImagePicker.  I think this issue might have to do with that I manually link react native (and most react native libraries), and I don't install it as a cocoapod.

**Comment by ivpusic** [comment]
tried it on device and it works for me. @theseansy I am also not using cocoapods to link react-native, just to download external dependencies and I have no problems.

**Comment by joeferraro** [comment]
I've been having a ton of issues both running on the device and archiving. Various errors ranging from missing libraries to "Native module cannot be null" when running in the simulator. I have followed the instructions several times without success.

**Comment by thanh-taro** [comment]
I have the same problem here when trying to run on device with cocoapod. It's ok when I use the other way. So weird.

**Comment by soroushchehresa** [comment]
this issue is happening when don't take completely step 2 of Post-install steps.
manual install not working and you can only use Cocoapods.
after install pod and take steps for resolve this error **open your-project.xcworkspace file with xcode and run project from here**.

NOTE:
your-project.xcworkspace file created after take step 2 of Post-install steps completely in ios folder of your project.
